packages/p2p/src/components/order-details/order-details.scss

```diff
@@ -56,6 +57,11 @@ $card-width: 456px;
         padding: 1rem 2.4rem;
     }
 
+    &--padding:last-of-type {
+        margin-bottom: 0px !important;
+        padding-bottom: 0px !important;
+    }
```

Comment:
Removed bottom margin and padding in last row to remove extra spaces
### Before
<img width="416" alt="Screenshot 2022-04-25 at 12 36 17 PM" src="https://user-images.githubusercontent.com/103016120/165021698-1f5ef981-7e55-4fad-9e3e-b887fbc38e3b.png">

### After
<img width="401" alt="Screenshot 2022-04-25 at 12 37 43 PM" src="https://user-images.githubusercontent.com/103016120/165021767-87703b6a-876d-4913-90c9-ee3caa832edd.png">